Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node
+
+WORKDIR /opt/first-blog-mern
+
+COPY . /opt/first-blog-mern 
```

Comment:
`COPY . .` will work

Reply:
Oh i see. Copy takes in <src> and <dest> arguements. Changed.